src/AppScopeTrait.php

```diff
@@ -22,11 +24,11 @@ trait AppScopeTrait
     public $_appScopeTrait = true;
 
     /**
-     * Always points to current Application.
+     * Always points to current application.
      *
-     * @var \atk4\ui\App
+     * @var App
      */
-    public $app;
+    private $app;
```

Comment:
because of trait design, private prop will still be accessible from the class where used

we may want to rename to `_app`, but then any assigment will not throw - but it can be solved by checking `->app` inside init or in the getter/setter methods in this trait